.github/workflows/dav1d.yml

```diff
@@ -49,32 +49,58 @@ jobs:
       run: |
         cargo test --workspace --all-features
 
-  windows-tests-gnu:
+  windows-tests-msvc:
 
     runs-on: windows-latest
 
-    env:
-      MSYSTEM: MINGW64
-      MSYS2_PATH_TYPE: inherit
-
     steps:
     - uses: actions/checkout@v3
+    - uses: ilammy/msvc-dev-cmd@v1
+
+    - name: Install nasm
+      uses: ilammy/setup-nasm@v1
 
-    - name: Install Rust Windows gnu
+    - name: Install Rust stable MSVC
       uses: actions-rs/toolchain@v1
       with:
         profile: minimal
-        toolchain: stable-x86_64-pc-windows-gnu
+        toolchain: stable-x86_64-pc-windows-msvc
         override: true
 
-    - name: Install msys2 packages
-      uses: msys2/setup-msys2@v2
+    - name: Install Python 3.9
+      uses: actions/setup-python@v4
       with:
-        msystem: MINGW64
-        install: mingw-w64-x86_64-pkgconf mingw-w64-x86_64-dav1d
-        update: true
+        python-version: '3.9'
+
+    - name: Install pip packages
+      run: |
+        pip install -U pip
+        pip install -U wheel setuptools
+        pip install -U meson ninja
+
+    - name: Setting up environment
+      run: |
+        echo "PKG_CONFIG=c:\build\bin\pkg-config.exe" >> $GITHUB_ENV
+        echo "PKG_CONFIG_PATH=C:\build\lib\pkgconfig" >> $GITHUB_ENV
+        echo "C:\build\bin" >> $GITHUB_PATH
+      shell: bash
```

Comment:
Can we move `shell: bash` after `name` field?

Reply:
Sure, not sure if that works though. Let's try. All examples I saw had it in the end.

Reply:
Works apparently too